ci-operator/step-registry/openshift/upgrade/vsphere/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- smarterclayton
+- wking
```

Comment:
I know nothing about vSphere.  If this just "folks who care about generic update stuff are now on the hook for the vSphere workflows, because nobody cares specifically about the vSphere/update intersection", that's ok, but I'd rather you make that more clear with a symlink (see #12832, which I just filed).  If there are actually folks that care about vSphere update CI (you?  Anyone else?), then I recommend creating a new `OWNERS` file pointing at those folks, so folks who want to talk about vSphere updates failing at some future point have an easier time finding interested parties.

Reply:
Yeah I will change the owners